mesonbuild/backend/vs2010backend.py

```diff
@@ -656,6 +657,13 @@ def create_basic_project(self, target_name, *,
         #   "The build tools for v142 (Platform Toolset = 'v142') cannot be found. ... please install v142 build tools."
         # This is extremely unhelpful and misleading since the v14x build tools ARE installed.
         ET.SubElement(root, 'Import', Project=r'$(VCTargetsPath)\Microsoft.Cpp.props')
+        ext_settings_grp = ET.SubElement(root, 'ImportGroup', Label='ExtensionSettings')
```

Comment:
Wouldn't it better to add this node only if `masm_type`?

Reply:
I kept this in case we potentially need further `BuildCustomizations`, or make vs find where to properly add the element if you tick that build customization box in the GUI.